spec/lib/stripe/credit_card_cloner_spec.rb

```diff
@@ -5,76 +5,83 @@
 
 module Stripe
   describe CreditCardCloner do
-    describe "#find_or_clone" do
-      include StripeStubs
+    let!(:user) { create(:user, email: "apple.customer@example.com") }
+    let!(:enterprise) { create(:enterprise) }
 
-      let(:credit_card) { create(:credit_card, user: create(:user)) }
-      let(:stripe_account_id) { "abc123" }
+    let(:secret) { ENV.fetch('STRIPE_SECRET_TEST_API_KEY', nil) }
 
-      let(:cloner) { Stripe::CreditCardCloner.new(credit_card, stripe_account_id) }
+    describe "#find_or_clone", :vcr, :stripe_version do
+      include StripeStubs
```

Comment:
Do we still need this ? I though we were removing stubs here.

Reply:
Indeed we don't - left forgotten. Thank you.